wordpress-mobile/WordPress-iOS
# Change navigation bar and top tabs background color in Reader and Notifications

## Requirement
> The background colour on dark mode for the navigation bar is different on Reader and Notifications. Notifications is the same as Home. So I think it is reader that is incorrect. See image below. Also the color is different between the navigation bar and the top tabs, but that's less of an issue to me. - @osullivanchris 

Source:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/20272#pullrequestreview-1328234102
- p1678204868515719/1678201408.773759-slack-C04SFL0RP51

## Description

This PR adds the following changes:
- The **Reader's** navigation bar background color changed to match **Notifications** and **Home** navigation bar color.
- [Jetpack-only] The top tabs background color changed to match the navigation bar color. 

| Screen | Before | After |
| ------- | ------- | ---- |
| Reader | <img width="375" alt="before-reader" src="https://user-images.githubusercontent.com/9609223/223652190-dd5f7524-9d06-449d-98a4-8b36436738e0.png"> | <img width="378" alt="after-reader" src="https://user-images.githubusercontent.com/9609223/223652174-bdddedb4-2fd7-4b93-9998-ff8ca7f3d6a4.png"> |
| Reader > Following | <img width="380" alt="before-reader-following" src="https://user-images.githubusercontent.com/9609223/223652194-cd199846-f63d-45c8-9ba1-503558369a44.png"> | <img width="378" alt="after-reader-following" src="https://user-images.githubusercontent.com/9609223/223652181-eb4f6271-467f-485c-8203-c6c4ad25d84a.png"> |
| Reader > Search | <img width="376" alt="before-reader-search" src="https://user-images.githubusercontent.com/9609223/223652196-4b7eb6d8-3496-4003-8420-a92fa6824112.png"> | <img width="376" alt="after-reader-search" src="https://user-images.githubusercontent.com/9609223/223652185-d5952523-374e-4a19-b58d-1d4685dab446.png"> |
| Notifications | <img width="375" alt="before-notifications" src="https://user-images.githubusercontent.com/9609223/223652188-7dda4dfe-d3ab-4f53-a4a6-20b57912f396.png"> | <img width="374" alt="after-notifications" src="https://user-images.githubusercontent.com/9609223/223652170-5a1987b9-a41c-4e3c-8b59-3d9ac5de7747.png"> |
| My Site > Stats | <img width="384" alt="before-stats" src="https://user-images.githubusercontent.com/9609223/223703264-450f74a1-1485-4b2a-98c2-d7d5ad54b78d.png"> | <img width="447" alt="after-my-site-stats" src="https://user-images.githubusercontent.com/9609223/223701209-4f911161-00e3-47cc-a1bb-8f6cf3cdf5ec.png"> |

## Test Instructions

1. Install Jetpack and log into your account.
2. Enable Dark Mode.
3. Repeat the following step for these screens:
   1. Reader tab
   2. Reader > Followed Topics / Sites ( the icon next to 🔍 button ) 
   4. Reader > Search  🔍
   5. Notifications tab
   6. My Site > Stats
3. **Verify** the top tabs background color matches the navigation bar background color
4. When on **Reader** screen, **Verify** the navigation bar background matches the one in **Notifications** screen.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
